### PR TITLE
Update members-list.md

### PR DESCRIPTION
**Pull request by ManuMBhat**

### CODE REVIEW

File: members-list.md [hunk 1]
```diff
@@ -2,15 +2,11 @@
...
 **About Me**:  `<-- Don't forget to add two spaces here`  
-Write a short, fun description about yourself. Don't overthink it and please be genuine and light-hearted.
-This isn't a flexing contest.
-Also don't be a cop-out by saying something like "I am happy/looking forward to be in PESOS".
-Tell us something we don't know.
-
+I am a newbie at open source and would be welcome to anything new to be learnt. I tend to get excited about lot of things and try to do many things(ik, not good!), but I do get most things accomplished(atleast that's what I believe).
 ## The list
 
 [Add your details over here] `← Don't remove this line`
```
Comment (tfidfwastaken) [inline members-list.md]: Add your details below this line^

**Comment by ManuMBhat** [comment]
Hey man
I'm not able to understand what to do
As I said I'm a newbie, sorry for the trouble

**Comment by tfidfwastaken** [comment]
Hey, don't worry about it for now in that case. Just stay on the slack group, I won't be removing you for the time being. When you come to our first meet-up, just tell me, I'll show you how to do this.

In the meantime just message me on the slack as to what part seemed unclear to you, I'm trying to figure out how to write my instructions in the clearest way possible and see what I can do to improve it.

I'll be closing this PR for now.

**Comment by ManuMBhat** [comment]
Thanks man,
I know your apartment mate Pranay really well, I could come over on tuesday
if you dont mind
Your instructions were clear, I misread it
Sorry for any inconvinence I may have caused.
Thanking You
Yours Sincerely
Manu M Bhat

**Comment by tfidfwastaken** [comment]
Hi, just meet me at college, that would be chill. 
Don't mind me deleting your comment because I don't want that possible information about my location shown in public :p

**Comment by ManuMBhat** [comment]
Peace
I understand

**Pull request closed without merging**